### PR TITLE
Add export for all types of grants

### PR DESCRIPTION
**Pull request by mattbee**
**What**  
Ability to export different grant types individually for easier processing.

**Why**  
To reduce admin when arranging payment for grants

### CODE REVIEW

File: lib/usecases/patchApplications.js [hunk 1]
```diff
@@ -1,16 +1,33 @@
...
+  }
+
   const query = `
     WITH exported AS
     (
         UPDATE
             application_assessment
         SET
-            application_state_id = 16 -- Exported for Payment
+            ${grant_type}_payment_exported = TRUE -- Mark as Exported for Payment
```
Comment (yusufsheiqh) [inline lib/usecases/patchApplications.js:24]: Do we want to set the application status to `exported for payment` as well?

**Review by yusufsheiqh — COMMENTED** [review]
I'm not sure if it is the way I have set up my local development environment, but all of the applications are showing the `exported` tag next to `LRSG (Closed businesses)`.

File: lib/usecases/applicationDetails.js [hunk 2]
```diff
@@ -26,6 +26,10 @@ export default async ({ clientGeneratedId }) => {
       lrsgClosedBusinessesAmount: application.lrsg_closed_businesses_amount,
       lrsgSectorAmount: application.lrsg_sector_amount,
       lrsgOpenAmount: application.lrsg_open_amount,
+      lrsgClosedBusinessesPaymentExported:
+        application.lrsg_closed_businesses_amount,
```
Comment (yusufsheiqh) [inline lib/usecases/applicationDetails.js:30]: We want the export here, not the actual amount.

```suggestion
        application.lrsg_closed_businesses_payment_exported,
```

File: lib/usecases/patchApplications.js (hunk 1 above)
Comment (mattbee) [reply lib/usecases/patchApplications.js:24]: No, I discussed this with Miranda and she just wanted status left alone.

**Review by yusufsheiqh — APPROVED** [review]

**Pull request merged**